packages/rsocket-ws-gateway/src/utils.ts

```diff
@@ -0,0 +1,33 @@
+export const packError = (e) => {
+  try {
+    const message = JSON.stringify(e);
```

Comment:
are you sure you need to change the data in-order to send it? i think rsocket do it for you